src/main/java/internet/shop/dao/impl/ManufacturerDaoImpl.java

```diff
@@ -1,46 +1,119 @@
 package internet.shop.dao.impl;
 
 import internet.shop.dao.ManufacturerDao;
-import internet.shop.db.Storage;
+import internet.shop.exception.NoConnectException;
 import internet.shop.lib.Dao;
 import internet.shop.model.Manufacturer;
+import internet.shop.util.ConnectionUtil;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.Optional;
 
 @Dao
 public class ManufacturerDaoImpl implements ManufacturerDao {
     @Override
     public Manufacturer create(Manufacturer manufacturer) {
-        Storage.addManufacturer(manufacturer);
+        String query = "INSERT INTO manufacturers (manufacturer_name, "
+                + "manufacturer_country) values (?, ?);";
+
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement preparedStatement = connection.prepareStatement(query,
+                        Statement.RETURN_GENERATED_KEYS)) {
+            preparedStatement.setString(1, manufacturer.getName());
+            preparedStatement.setString(2, manufacturer.getCountry());
+
+            preparedStatement.executeUpdate();
```

Comment:
```suggestion
                + "manufacturer_country) values (?, ?);";
        try (Connection connection = ConnectionUtil.getConnection();
                PreparedStatement preparedStatement = connection.prepareStatement(query,
                        Statement.RETURN_GENERATED_KEYS)) {
            preparedStatement.setString(1, manufacturer.getName());
            preparedStatement.setString(2, manufacturer.getCountry());
            preparedStatement.executeUpdate();
```
these empty lines are redundant, they don't improve readability but make the code longer

Reply:
thanks